.github/scripts/windows/test_task.bat

```diff
@@ -32,18 +30,21 @@ set PDO_MYSQL_TEST_PASS=%MYSQL_PWD%
 set PDO_MYSQL_TEST_HOST=%MYSQL_TEST_HOST%
 set PDO_MYSQL_TEST_PORT=%MYSQL_TEST_PORT%
 set PDO_MYSQL_TEST_DSN=mysql:host=%PDO_MYSQL_TEST_HOST%;port=%PDO_MYSQL_TEST_PORT%;dbname=test
-set TMP_MYSQL_BIN=C:\mysql\bin
-"%TMP_MYSQL_BIN%\mysql.exe" --host=%PDO_MYSQL_TEST_HOST% --port=%MYSQL_TEST_PORT% --user=%MYSQL_TEST_USER% --password=%MYSQL_TEST_PASSWD% -e "CREATE DATABASE IF NOT EXISTS test"
+set PATH=%PATH%;C:\mysql\bin
+mysql.exe --host=%PDO_MYSQL_TEST_HOST% --port=%MYSQL_TEST_PORT% --user=%MYSQL_TEST_USER% --password=%MYSQL_TEST_PASSWD% -e "CREATE DATABASE IF NOT EXISTS test"
 if %errorlevel% neq 0 exit /b 3
 
 rem setup PostgreSQL related exts
 set PGUSER=postgres
 set PGPASSWORD=Password12!
 rem set PGSQL_TEST_CONNSTR=host=127.0.0.1 dbname=test port=5432 user=postgres password=Password12!
+git checkout "./ext/pgsql/tests/config.inc"
 echo ^<?php $conn_str = "host=127.0.0.1 dbname=test port=5432 user=%PGUSER% password=%PGPASSWORD%"; ?^> >> "./ext/pgsql/tests/config.inc"
 set PDO_PGSQL_TEST_DSN=pgsql:host=127.0.0.1 port=5432 dbname=test user=%PGUSER% password=%PGPASSWORD%
-set TMP_POSTGRESQL_BIN=%PGBIN%
-"%TMP_POSTGRESQL_BIN%\createdb.exe" test
+set PATH=%PATH%;%PGBIN%
+psql -U postgres -c "DROP SCHEMA IF EXISTS test"
+psql -U postgres -c "CREATE SCHEMA IF NOT EXISTS test"
```

Comment:
you'll see this pattern copied with mysql. Basically the createdb command does not exist on my local install of postgres. This makes sure there is a schema and that it's fresh. It also uses path so if the path explicitly supplied did exist, then we can use that set of binaries. But respects user paths if not.